storage/mysql/log_storage.go

```diff
@@ -138,6 +137,10 @@ func (t *readOnlyLogTX) Rollback() error {
 	return t.tx.Rollback()
 }
 
+func (t *readOnlyLogTX) CheckDatabaseAccessible() error {
```

Comment:
I've reverted back to "CheckDatabaseAccessible", on the grounds of:
1. We have a few methods named like that
2. It better matches the interface name
3. *IsConnected* is not entirely true, as the storage doesn't necessarily keep an active connection (plus it may suggest a different kind of usage)
4. Al doesn't seem to mind it